server/grpc_service.go

```diff
@@ -3000,6 +3001,78 @@ func (s *GrpcServer) GetExternalTimestamp(ctx context.Context, request *pdpb.Get
 	}, nil
 }
 
+// ManuallyChangePeer implements gRPC PDServer.
+func (s *GrpcServer) ManuallyChangePeer(ctx context.Context, request *pdpb.ChangePeerRequest) (*pdpb.ChangePeerResponse, error) {
```

Comment:
Any input about the RPC name? @nolouch @rleungx

Reply:
Can we only report the purpose then PD generate operators. like call interface as `ReportDamagedPeer .

Reply:
Do u mean using `GetDamagedRegionsId`?

Not appropriate. The `DamagedRegionsId` just records the abnormal region, and it's used to delete all of the peers of this region.

But what we need now is "add / remove" a single peer.

Reply:
I mean the TiKV tells PD which peer is damaged or needs to be cleaned. then PD will log it(better with the source) and add an operator to add/remove it.

Do not directly guide PD on how to do it, tell PD your purpose, and then PD makes the decisions. it's prevents some unknow source from abusing it.

Reply:
It just needs a simple way to let PD trigger the operator on `add` or `remove` peer here. And we've already have Restful API used for `add` & `remove` peer, gRPC API is intended to be used as the same purpose here.